spec/factories/leagues.rb

```diff
@@ -3,4 +3,9 @@
     name "American League"
     dh true
   end
+
+  factory :national_league, class: League do
```

Comment:
Ideally we wouldn't need to create new factories downstream to avoid collisions but I think its okay to punt on untangling FactoryGirl associations for now in favor of working on getting the core logic of the app working.